File: Leetcode/1550 - Three Consecutive Odds/solution.py
class Solution:
    def threeConsecutiveOdds(self, arr: List[int]) -> bool:
        if (len(arr) < 3):
            return False
        for idx in range(len(arr) - 2):
            if ((arr[idx] % 2) and (arr[idx + 1] % 2) and (arr[idx + 2] % 2)):
                return True
        return False

File: Leetcode/1550 - Three Consecutive Odds/solution.js
var threeConsecutiveOdds = function (arr) {
    if (arr.length < 3) {
        return false;
    }
    for (let idx = 0; idx <= (arr.length - 3); idx++) {
        if ((arr[idx] % 2) && (arr[idx + 1] % 2) && (arr[idx + 2] % 2)) {
            return true;
        }
    }
    return false;
};